dart_conf_flutter/lib/src/common.dart

```diff
@@ -8,9 +8,9 @@ final ThemeData _theme = new ThemeData(primarySwatch: Colors.blue);
 
 ThemeData get appTheme => _theme;
 
-final String loremIpsum =
-    'Lorem ipsum dolor sit amet, consectetur adipiscing elit. '
-    'Fusce laoreet magna nunc, nec volutpat mi lobortis non.';
+//final String loremIpsum =
```

Comment:
I'd just delete instead of commenting out.

Reply:
done